packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -182,8 +182,10 @@ const useTabs = () => {
           (childTab) => childTab.href !== "/settings/security/two-factor-auth"
         );
         return { ...tab, children: filtered };
-      } else if (tab.href === "/settings/developer" && !!orgBranding) {
-        const filtered = tab?.children?.filter((childTab) => childTab.name !== "admin_api");
+      } else if (tab.href === "/settings/developer") {
+        const filtered = tab?.children?.filter((childTab) =>
+          childTab.name === "admin_api" ? isOrgAdminOrOwner : true
+        );
```

Comment:
The issue describes that Only display this to organization customers. SO why the checks for isOrgAdminOrOwner?

Reply:
by "Admin API" i inferred as only for admin, updated.
true was to allow other childtabs to pass through filter in that case.

Reply:
Hey Vijay, talk to the team. Yes, organization customers mean it will only be visible to the org admin/owner. So your previous check was correct, but code needs to improve a bit. Let me update it.

Reply:
ok, updated